bot/cogs/queue.py

```diff
@@ -141,12 +118,18 @@ def all_ready(reaction, user):
         else:  # Everyone readied up
             # Attempt to make teams and start match
             await ready_message.clear_reactions()
+            guild_data = await self.bot.db_helper.get_guild(ctx.guild)
+            team_method = guild_data['team_method']
```

Comment:
Perhaps convert this value to lowercase before we run it through the if statements.

Reply:
It's based on a custom enum data type in the DB. Will always come out lower case. When I implement a way to set it I'll make sure it gets put in lowercase (and would require some sanitization pretty sure).

Reply:
Yeah that's good, I just want to make sure if this ever becomes something a user can set then we're covering all bases :+1: